README.md

```diff
@@ -6,168 +6,171 @@
 
 </div>
 
-#### For frontend of this repo move to [this](https://github.com/codeuino/frontened) repository.
+# About
 
-<div align='center'>
-<img style="border:1px black solid;" height=165px src="READMEassets/signup_community.png">
-<img style="border:1px black solid;" height=165px src="READMEassets/landing.gif">
-<img style="border:1px black solid;" height=165px src="READMEassets/signup_user.png">
-<img style="border:1px black solid;" height=165px src="READMEassets/feeds.png">
-<img style="border:1px black solid;" height=165px src="READMEassets/edit_profile.png">
-<img style="border:1px black solid;" height=165px src="READMEassets/dashboard.png">
-</div>
+Being inspired by the Cornucopia of various social hub this project has been developed taking into consideration about open source.
 
+Well, this is an Open Source Social networking hub which acts as a bridge between various Developers, Organisations and Open Source aspirants to elaborate on various things like Projects, Events, Discussion on various researches, Scholarships, Coding release and various other things updates.
 
-What is Donut?
---------
-Being inspired by the Cornucopia of various social hub this project has been developed taking into consideration about open source.
- 
- Well, this is an Open Source Social networking hub which acts as a bridge between various Developers, Organisations and Open Source aspirants to elaborate on various things like #Projects, #Events, #Discussion on various researches, #Scholarships, #Coding release and various other things updates. 
+The major priority of this project has been that this platform allows users to make their project "Open Sourced" and released them under various open source Organisations, experts which hold up a ring plate on this portal.
+
+This platform also makes users introduce and develops various solutions in the form of FOSS software to publish them for public use by integrating them with their social cause. Moreover, this project can be downloaded by any user, organization and can be used by them in their own custom way, making it run on their servers.
+
+### Frontend Technology Stack
+
+- CSS: Styling web pages, html files
+- Javascript: Primary programing language
+- ReactJS: Javascript library for building User Interfaces
+- Bootstrap: Styling framework.
+
+Learn more about the donut platform at the [official donut documentation](https://docs.codeuino.org/donut-social-networking-platform/).
+
+---
+
+# Setting up the work environment.
+
+Few things are required before setting up the project locally,
+
+- **A Web Browser** - You can install any browser as per your preference e.g. Internet Explorer,Chrome, FireFox, Safari, Opera etc. JavaScript works on any web browser on any OS.
```

Comment:
Actually, I think Internet Explorer, Safari, or Opera are not good choices for the work environment as the contributor will need react-dev-tools and redux-dev-tools. We could make good use of this section to provide links to those extensions.